_data/accessibility-tests/time-picker.yml

```diff
@@ -0,0 +1,73 @@
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.4
+  - summary: Color alone is not used to convey information.
+    summary_additional: When viewing the time picker, you do not have to rely on color alone to understand information.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.1
+  - summary: Instructions for the time picker are clear.
+    summary_additional: When you view the time picker, you find clear, simple instructions on how to select a time and what type of time format is expected.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.3.2
+# Zoom/screen magnification tests
+  - summary: Time picker elements are visible and functional when screen is magnified.
+    summary_additional:  When you zoom in to 200%, the time picker is still readable, legible and properly sized.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.4
+  - summary: Using zoom does not obstruct the time picker.
+    summary_additional: When you zoom to 400%, you can see the time picker and options. It does not overlap with other content and no scrolling is needed.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Keyboard can operate the time picker.
+    summary_additional: When you navigate through the time picker with a keyboard, you can interact with every element inside it.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.1
+  - summary: Focus indicator is clearly visible.
+    summary_additional: When you use a keyboard to navigate through the time picker, there will be a visible outline or other clear indication around each element where the the focus is.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.7
+  - summary: Tabbing into the time picker doesn't trigger a change of context.
+    summary_additional: When you use a keyboard to access the time picker, there is no change until you activate with enter key or spacebar.
+    test_status: conditional
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.1
+# Screen reader tests
+  - summary: Screen reader announces elements to match visual order on screen.
+    summary_additional: When you use a screen reader to tab into the time picker, elements are announced in the same order that they appear on the page.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.2
+  - summary: Screen reader announces all instructions.
+    summary_additional: When you use a screen reader and navigate through a time picker, you hear every instruction for how to select a time and for any options available.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 3.3.2
```

Comment:
**Issue (blocking):** There was a typo in the [checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?gid=97481116#gid=97481116) 🔒 which listed two `3.3.2`

This should be listed as `1.3.1`.

```suggestion
  - summary: Screen reader announces all instructions.
    summary_additional: When you use a screen reader and navigate through a time picker, you hear every instruction for how to select a time and for any options available.
    test_status: pass
    test_type: screen_reader
    version_tested: 3.8.2
    wcag_criterion: 1.3.1
```

**Note:** I've updated the spreadsheet to correct the criterion number and move it in numerical order of the other testing items

Reply:
Resolved in [32cded2](https://github.com/uswds/uswds-site/pull/2921/commits/32cded2a0e92ea837065d2b62d034176cc77ce89)